geomesa-hbase/geomesa-hbase-tools/src/main/scala/org/locationtech/geomesa/hbase/tools/HBaseDataStoreCommand.scala

```diff
@@ -21,9 +24,35 @@ trait HBaseDataStoreCommand extends DataStoreCommand[HBaseDataStore] {
   override def params: HBaseParams
 
   override def connection: Map[String, String] = {
+
+    val HBASE_SITE_FILE_NAME = "hbase-site.xml"
+
+    def extractZookeepers(site: Node): String = {
```

Comment:
would it make sense to extract this with a hadoop Configuration object instead? I believe you could do something like:
```
val conf = new Configuration()
conf.addResources(foo)
conf.getString("hbase.zookeeper.quorom")
```
That should handle property substitution as well (if there are any)
@jahhulbert-ccri does that seem reasonable?

Reply:
ah, yeah that might be the more canonical method.

Reply:
yeah we shouldn't hard code hbase-site.xml bc that can be renamed and the real thing is that the param should be in the Configuration option...we already have some code around this here that needs to be respected when "creating" confs: https://github.com/locationtech/geomesa/blob/master/geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseConnectionPool.scala#L128

Also the line ```val conf = new Configuration()``` can take up to 1-2 seconds to do because it runs through the entire classpath (or used to at least). so you don't want to be creating new ones. I think we should figure out the interaction between this pr, the existing HbaseConnectionPool withPaths method, and the ticket https://geomesa.atlassian.net/browse/GEOMESA-2236

we are breaking in spark for the precise reason we call new Configuration() and lose the old one.